src/project/server/routes/api.js

```diff
@@ -0,0 +1,114 @@
+// Define your endpoints here (this is your "controller file")
+const express = require('express');
+const ItemManager = require('../services/item-manager.js')
+
+const todoRouter = express.Router();
+const itemManager = new ItemManager();
+
+todoRouter.get('/', async(req, res) => {
+    const {sort, filter} = req.query;
+    
+    let sortedList = [];
+
+    const sortObj = {
+        atoz:{field: 'itemName', order:'ASC'},
+        ztoa:{field: 'itemName', order:'DESC'},
+        dtou:{field: 'status', order:'DESC'},
+        utod:{field: 'status', order:'ASC'},
+    }
+
+    if(sort){
+        sortedList = await itemManager.orderData(sortObj[sort].field, sortObj[sort].order)
+    }
+    else if(filter){
```

Comment:
Moreover things like sort and filter of this kind where the default is to show all the todos, meaning you have the full list in the client before anything is filtered it is better to filter / sort in the client rather than making another server request since you already have everything you need in the client